staker/fast_confirm.go

```diff
@@ -0,0 +1,249 @@
+package staker
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/offchainlabs/nitro/solgen/go/contractsgen"
+	"github.com/offchainlabs/nitro/solgen/go/rollupgen"
+	"github.com/offchainlabs/nitro/staker/txbuilder"
+	"github.com/offchainlabs/nitro/util/headerreader"
+)
+
+type FastConfirmSafe struct {
+	rollupAddress             common.Address
+	safe                      *contractsgen.Safe
+	owners                    []common.Address
+	threshold                 uint64
+	fastConfirmNextNodeMethod abi.Method
+	builder                   *txbuilder.Builder
+	wallet                    ValidatorWalletInterface
+	gasRefunder               common.Address
+	l1Reader                  *headerreader.HeaderReader
+	fastConfirmApprover       common.Address
+}
+
+func NewFastConfirmSafe(
+	callOpts bind.CallOpts,
+	rollupAddress common.Address,
+	fastConfirmSafeAddress common.Address,
+	builder *txbuilder.Builder,
+	wallet ValidatorWalletInterface,
+	gasRefunder common.Address,
+	l1Reader *headerreader.HeaderReader,
+	fastConfirmApprover common.Address,
```

Comment:
Isn't `fastConfirmApprover` the same as `wallet.Address()`? I don't think we should need a separate config for this.

Reply:
Done.